### DISCUSSION TITLE
not able to reinstall code server

### DISCUSSION THREAD

**Discussion by ct24p** [coder/code-server#5891] [General]
1) Installed code-server-4.8.3
2) Removed it by "rm -rf ~/.local/share/code-server ~/.config/code-server"
3) Tried to re-install code-server-4.9.0
4) errors: /usr/lib/code-server/.... conflicts

tried step 5 below, still issue (on Redhat server 7.9)
5) removed /usr/lib/code-server folder

Could you please help?

**Comment by jsjoeio** [comment]
Hmm...Makes me wish we had a flag `--force` to overwrite existing code-server installs with the install script.

So you did Step 5, can you output the logs when you try installing again? And how are you installing it?

**Comment by ct24p** [comment]
Thanks.
followed instructions below:
    curl -fOL https://github.com/coder/code-server/releases/download/v$VERSION/code-server-$VERSION-amd64.rpm
    sudo rpm -i code-server-$VERSION-amd64.rpm

error messages: a lot of "/usr/lib/code-server/... 4.8.3 already exist ...." , one thread indicating related to rpm _db cache

**Reply by jsjoeio** [reply]
Have you tried using the install script instead? I think that will use rpm and handle upgrading:

```shell
curl -fsSL https://code-server.dev/install.sh | sh
```
